website/integrations/sources/discord/index.md

```diff
@@ -135,6 +135,17 @@ guild_member_object = requests.get(
     }
 ).json()
 
+# The response for JSON errors is held within guild_member_object['code']
+# See: https://discord.com/developers/docs/topics/opcodes-and-status-codes#json
+# If the user isn't in the queried guild, it gives the somewhat misleading code = 10004.
+if "code" in guild_member_object:
+    if guild_member_object['code'] == 10004:
+        ak_message(f"User is not a member of {GUILD_NAME_STRING}.")
+    else:
+        ak_message(f"Error, check https://discord.com/developers/docs/topics/opcodes-and-status-codes#json for error code {guild_member_object['code']}.")
```

Comment:
For that it would probably make sense to create an Event in the log instead of telling the user information they can't really do anything with

```suggestion
        ak_create_event("discord_error", source=context['source'], code= guild_member_object['code']})
```

Reply:
Good idea! I'm not familiar with the event logging bits as yet so I'll defer here to your suggestions.